pkgs/development/ocaml-modules/cstruct/default.nix

```diff
@@ -1,20 +1,40 @@
-{stdenv, writeText, fetchurl, ocaml, ocplib-endian, sexplib_p4, findlib,
- async_p4 ? null, lwt ? null, camlp4}:
+{stdenv, buildOcaml, fetchFromGitHub, writeText,
+ ocaml, ocplib-endian, sexplib_p4, findlib, ounit, camlp4,
```

Comment:
Thank you! Implementing your suggestions seems to fix the build problem with enabled async support I encountered.